client/sites-dashboard/components/sites-search.ts

```diff
@@ -4,11 +4,15 @@ import styled from '@emotion/styled';
 export const SitesSearch = styled( Search )( {
 	'--color-surface': 'var( --studio-white )',
 
-	height: '42px',
+	height: '42px !important',
+
 	overflow: 'hidden',
 	border: '1px solid #c3c4c7',
 
 	'@media screen and (min-width: 660px)': {
-		flex: '0 1 390px',
+		flex: '0 1 390px !important',
+
+		height: 'auto !important',
```

Comment:
Another way of doing that would be increasing the specificity of this component, without using `!important`:

```ts
'&.search-component': {
	height: '42px',

	overflow: 'hidden',
	border: '1px solid #c3c4c7',

	'@media screen and (min-width: 660px)': {
		flex: '0 1 390px',

		height: 'auto',
		alignSelf: 'stretch',
	},
},
```

But IMO this is harder to read and reason about. Plus, it's also dependent on the `search-component` class name, which is fragile because it may change without warning.

Reply:
I recognise this issue also. CSS-in-JS for the win, the styles being removed when the component is unmounted is a big win.